Extensions/TileMap/JsExtension.js

```diff
@@ -563,55 +563,6 @@ module.exports = {
      * This is called to update the PIXI object on the scene editor
      */
     RenderedTileMapInstance.prototype.update = function () {
-      const tilemapJsonFile = this._associatedObject
-        .getProperties(this.project)
-        .get('tilemapJsonFile')
-        .getValue();
-      if (this._pixiObject.tilemapJsonFile !== tilemapJsonFile)
-        this._pixiObject.tilemapJsonFile = tilemapJsonFile;
-
-      const tilesetJsonFile = this._associatedObject
-        .getProperties(this.project)
-        .get('tilesetJsonFile')
-        .getValue();
-      if (this._pixiObject.tilesetJsonFile !== tilesetJsonFile)
-        this._pixiObject.tilesetJsonFile = tilesetJsonFile;
-
-      const tilemapAtlasImage = this._associatedObject
-        .getProperties(this.project)
-        .get('tilemapAtlasImage')
-        .getValue();
-      if (this._pixiObject.tilemapAtlasImage !== tilemapAtlasImage)
-        this._pixiObject.tilemapAtlasImage = tilemapAtlasImage;
-
-      const displayMode = this._associatedObject
-        .getProperties(this.project)
-        .get('displayMode')
-        .getValue();
-      if (this._pixiObject.displayMode !== displayMode)
-        this._pixiObject.displayMode = displayMode;
-
-      const layerIndex = this._associatedObject
-        .getProperties(this.project)
-        .get('layerIndex')
-        .getValue();
-      if (this._pixiObject.layerIndex !== layerIndex)
-        this._pixiObject.layerIndex = layerIndex;
```

Comment:
This does update in the ide when you change it, so does any of the other non animation related values. Does that still work in the ide when you remove them here

Reply:
Edit: ah sorry, just read the comment about putting them on pixi object. Yeah that is totally ok. I sometimes get confused with the extension api approach of updating in the ide